app/src/main/java/com/tari/android/wallet/ui/fragment/onboarding/CreateEmojiIdFragment.kt

```diff
@@ -0,0 +1,347 @@
+        val fadeOut = ValueAnimator.ofFloat(1f, 0f)
+        fadeOut.duration = Constants.UI.CreateEmojiId.shortAlphaAnimDuration
+        fadeOut.addUpdateListener { valueAnimator: ValueAnimator ->
+            val alpha = valueAnimator.animatedValue as Float
+            justSecDescText.alpha = alpha
+            justSecTitle.alpha = alpha
+        }
+
+        fadeOut.addListener(object : AnimatorListenerAdapter() {
+            override fun onAnimationEnd(animation: Animator?, isReverse: Boolean) {
+                super.onAnimationEnd(animation, isReverse)
+                checkMarkAnim.visibility = View.VISIBLE
+
+            }
+        })
+        fadeOut.start()
+        checkMarkAnim.playAnimation()
+    }
+
+    private fun startWhiteBackGroundAnimation() {
+        val whiteBgViewAnim: ObjectAnimator =
+            ObjectAnimator.ofFloat(
+                whiteBgView,
+                View.TRANSLATION_Y,
+                -whiteBgView.height.toFloat(),
+                0f
+            )
+
+        whiteBgViewAnim.startDelay = Constants.UI.CreateEmojiId.whiteBgAnimDelayMs
+        whiteBgViewAnim.duration = Constants.UI.CreateEmojiId.whiteBgAnimDurationMs
+        whiteBgViewAnim.interpolator = EasingInterpolator(Ease.SINE_OUT)
+        whiteBgViewAnim.addListener(object : AnimatorListenerAdapter() {
+            override fun onAnimationEnd(animation: Animator?) {
+                super.onAnimationEnd(animation)
+                startInitHelloTextAnimation()
+            }
+
+            override fun onAnimationStart(animation: Animator?) {
+                super.onAnimationStart(animation)
+                smallGem.visibility = View.VISIBLE
+                whiteBgView.visibility = View.VISIBLE
+            }
+        })
+        whiteBgViewAnim.start()
+    }
+
+    private fun startInitHelloTextAnimation() {
+        val offset = -helloText.height.toFloat()
+        val helloTextAnim: ObjectAnimator =
+            ObjectAnimator.ofFloat(helloText, View.TRANSLATION_Y, 0f, offset)
+
+        helloTextAnim.duration = Constants.UI.CreateEmojiId.helloTextAnimDurationMs
+        helloTextAnim.interpolator = EasingInterpolator(Ease.QUINT_OUT)
+
+        helloTextAnim.addListener(object : AnimatorListenerAdapter() {
+            override fun onAnimationStart(animation: Animator?) {
+                super.onAnimationStart(animation)
+                helloTextBackView.visibility = View.VISIBLE
+                helloText.visibility = View.VISIBLE
+                justSecBackView.visibility = View.VISIBLE
+            }
+
+            override fun onAnimationEnd(animation: Animator?) {
+                super.onAnimationEnd(animation)
+                uiHandler.postDelayed({
+                    startSecondViewTextAnimation()
+                }, Constants.UI.CreateEmojiId.viewChangeAnimDelayMs)
+            }
+        })
+        helloTextAnim.start()
+    }
+
+    private fun startTariViewAnimation() {
```

Comment:
Actually I really think this animation belongs to the previous fragment - it's part of that screen, not this one.